pom.xml

```diff
@@ -496,7 +496,7 @@
     <!-- Enable forbiddenapis plugin, do disable set to none -->
     <api.check.phase>process-classes</api.check.phase> <!-- forbidden-apis dislikes TestValidationUtils #validateUtilityClass -->
     <message.xml.version>31</message.xml.version>
-    <repository.settings.version>13</repository.settings.version>
+    <repository.settings.version>14-SNAPSHOT</repository.settings.version>
```

Comment:
Can be changed to 14 now. :)

Reply:
Changed in the main pom file. Have you handled the changes in bitrepository-core/pom.xml already?

Reply:
I have now :)